packages/textmate-grammars/src/browser/shell.ts

```diff
@@ -21,7 +21,7 @@ import { injectable } from 'inversify';
 export class ShellContribution implements LanguageGrammarDefinitionContribution {
 
     readonly id = 'shell';
-    readonly scopeName = 'source.sh';
+    readonly scopeName = 'source.shell';
```

Comment:
could you check for `LessContribution.scopeName` too?

Reply:
Did that [here](https://github.com/theia-ide/theia/pull/3131) already.